src/workloads/scale/UpdateMillionDocsInTxn.yml

```diff
@@ -0,0 +1,78 @@
+# This workload is developed to test the amount of time it takes to update a million documents in
+# a single replica set transaction. At the moment, the average time taken raises above the default
+# limit, so until we add a way to manually increase a transaction's lifetime, we must raise the
+# lifetime of all transactions.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+
+Actors:
+- Name: Loader
+  Type: Loader
+  Threads: 1
+  Phases:
+  - ExternalPhaseConfig:
+      Path: &PhasePath ../../phases/scale/DesignDocWorkloadPhases.yml
+      Key: Loader
+      Parameters:
+        CollectionCount: 1
+        DocumentCount: 1e6
+        BatchSize: 100000
+    Repeat: 1
+    Threads: 1
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: RaiseTransactionLifetimeLimit
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Operation:
+      OperationName: RunCommand
+      OperationCommand:
+        setParameter: 1
+        transactionLifetimeLimitSeconds: 120
+  - *Nop
+  - Repeat: 1
+    Operation:
+      OperationName: RunCommand
+      OperationCommand:
+        setParameter: 1
+        transactionLifetimeLimitSeconds: 60
```

Comment:
this 1 operation will happen concurrently with the `UpdateDocumentsInTransaction` in Phase index 1. Do you want this to happen before you start that Phase? If so you could have a separate Phase that just does this `setParameter` operation.

Reply:
It seems to me like the first phase is just the loader and the admin command, while the `UpdateDocumentsInTransaction` is a no-op. Am I reading this wrong?